tests/fail/data_race/atomic_write_na_read_race2.rs

```diff
@@ -16,9 +16,7 @@ pub fn main() {
     let b = &mut a as *mut AtomicUsize;
     let c = EvilSend(b);
     unsafe {
-        let j1 = spawn(move || {
-            *(c.0 as *mut usize)
```

Comment:
If we add a trailing `;` here, does rustfmt keep the braces? I think I'd prefer that, for symmetry with the other `spawn`.